app.py

```diff
@@ -29,11 +30,34 @@
 from thoth.storages import AnalysisResultsStore
 from thoth.storages import SolverResultsStore
 
+from prometheus_client import CollectorRegistry, Gauge, Counter, push_to_gateway
+
+prometheus_registry = CollectorRegistry()
+
 __version__ = "0.1.0"
 __component_version__ = f"{__version__}+storages{__thoth_storages_version__}"
 
 init_logging()
 _LOGGER = logging.getLogger("thoth.document_sync")
+_THOTH_DEPLOYMENT_NAME = os.environ["THOTH_DEPLOYMENT_NAME"]
+_THOTH_METRICS_PUSHGATEWAY_URL = os.getenv("PROMETHEUS_PUSHGATEWAY_URL")
+
+# Metrics Document sync
+_METRIC_INFO = Gauge(
```

Comment:
Would it make sense to count each document type? It could help us narrow down documents that failed to sync.

Reply:
Yes that is a great idea!

Reply:
Added in https://github.com/thoth-station/document-sync-job/pull/6/commits/566b4593121924ea923462677d64d7ef2f57df2f